go/database/mpt/live_trie_test.go

```diff
@@ -960,3 +960,34 @@ func BenchmarkValueInsertionInFileTrie(b *testing.B) {
 		})
 	}
 }
+
+func BenchmarkLiveTrie_WithHighNumberOfDirtyNodes_Flush(b *testing.B) {
+	// running this test for all configurations at once takes too much time
+	// hence we use only the latest one
+	config := S5ArchiveConfig
+	// largest nodes have ~512 bytes - we need to create a cache with 18 GB cache,
```

Comment:
I checked yesterday, while investigating memory issues reported by Jan: the largest nodes actually have 944 bytes.

Reply:
It make sense. Grafana showed memory consumption of <30GiB. Thanks for the heads up!